DataFormats/Scouting/src/classes_def.xml

```diff
@@ -41,11 +41,42 @@
   <class name="Run3ScoutingVertex" ClassVersion="3">
       <version ClassVersion="3" checksum="316446070"/>
   </class>
-  <class name="Run3ScoutingElectron" ClassVersion="5">
+  <class name="Run3ScoutingElectron" ClassVersion="6">
       <version ClassVersion="3" checksum="1086011373"/>
       <version ClassVersion="4" checksum="1250202632"/>
       <version ClassVersion="5" checksum="2230390721"/>
-  </class>
+      <version ClassVersion="6" checksum="2815634332"/>
+  </class>
+
+  <!-- Adding ioread rules for backwards compatibility -->
+  
+  <ioread sourceClass="Run3ScoutingElectron" version="[3-5]" 
+	  source="float d0_;" 
+	  targetClass="Run3ScoutingElectron" target="trkd0_">
+    <![CDATA[
+	     trkd0_.clear();
+	     trkd0_.reserve(1);
```

Comment:
The `reserve(1)` is practically unnecessary, since only one element is inserted.